src/commonMain/kotlin/com/wizbii/cinematic/journey/presentation/util/NumberFormat.kt

```diff
@@ -2,6 +2,20 @@ package com.wizbii.cinematic.journey.presentation.util
 
 import kotlin.math.roundToLong
 
+fun Number.formatHumanReadable(): String {
+    val suffix = arrayOf("", "K", "M", "G", "T", "P", "E", "Z", "Y")
+
+    var suffixIndex = 0
+    var value = toDouble()
+
+    while (value > 1000 && suffixIndex < suffix.size - 1) {
```

Comment:
`suffix.size - 1` -> `suffix.lastIndex`
We could also use `suffixIndex in suffix.indices`

Reply:
Updated